components/SkillsGrid.jsx

```diff
@@ -0,0 +1,17 @@
+export default function SkillsGrid({ skilldata }) {
+  return (
+    <div className="cmp-skills-grid">
+      <h2 className="cmp-skills-grid__heading">Skills</h2>
+      <div className="obj-skills-grid">
+        {skilldata && skilldata.map((data) => (
```

Comment:
https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

This is good! This was shared in Dev the other day and I thought it was thought-provoking.